phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -76,6 +79,11 @@ public RepositoryDTO fetchInstallableApplications() {
         }
     }
 
+    @Override
+    public boolean isSafe() {
```

Comment:
Basically, all repositories instance of a given class have not necessarely the same safeness state. So annotations won't be powerful enough.
I'm not even sure that isSafe method is the right solution.